Sources/SentryCrash/Recording/SentryCrashReportStore.c

```diff
@@ -107,7 +107,7 @@ getReportCount()
     }
     struct dirent *ent;
     while ((ent = readdir(dir)) != NULL) {
-        if (getReportIDFromFilename(ent->d_name) > 0) {
+        if (ent->d_type != 4 && getReportIDFromFilename(ent->d_name) > 0) {
```

Comment:
`m`: Please create a constant for `4`. I have no idea what it means.

Reply:
I can do that ;)

Reply:
Or even better, use the default constant meant for this.